app/views/layouts/application.html.haml

```diff
@@ -36,8 +36,16 @@
             %li
               %a.active{:href => "/"} Home
           %ul.nav.navbar-nav.navbar-right.musikore-navbar-item
-            %li
-              %a{:href => "/login"} Log in
+            - if user_signed_in?
+              %li
+                = current_user.username
+              %li
+                = link_to "Edit Profile", "/edit"
+              %li
+                = link_to "Log out", destroy_user_session_path, method: :delete
```

Comment:
@starkshaw similar to this. To find all routes run `rake routes` and get the name from there and add `_path` in the end of that named route.